accounts/templatetags/backend_utils.py

```diff
@@ -0,0 +1,81 @@
+import re
+
+from django import template
+
+from social.backends.oauth import OAuthAuth
+
+
+register = template.Library()
+
+name_re = re.compile(r'([^O])Auth')
+
+
+@register.filter
+def backend_name(backend):
+    name = backend.__class__.__name__
+    name = name.replace('OAuth', ' OAuth')
+    name = name.replace('OpenId', ' OpenId')
+    name = name.replace('Sandbox', '')
+    name = name_re.sub(r'\1 Auth', name)
+    return name
+
+
+@register.filter
+def backend_class(backend):
+    return backend.name.replace('-', ' ')
+
+
+@register.filter
+def icon_name(name):
```

Comment:
This seems unnecessary - since we are not using most of these services. I think any reference to other social auth services should be removed, so there is no ambiguity in why the accounts app exists and what it's function is.